src/transformers/models/mpnet/modeling_tf_mpnet.py

```diff
@@ -195,7 +196,6 @@ def __init__(self, config, **kwargs):
             )
 
         self.num_attention_heads = config.num_attention_heads
-        assert config.hidden_size % config.num_attention_heads == 0
```

Comment:
This one is deleted with no new test, there should be an if/error here.

Reply:
@sgugger I deleted it because there is already a check right above it. Please see [here](https://github.com/anmolsjoshi/transformers/blob/controlFlowAssertions/src/transformers/models/mpnet/modeling_tf_mpnet.py#L192-L196)